pkg/tsdb/cloudmonitoring/resource_handler.go

```diff
@@ -41,7 +41,8 @@ func getGCEDefaultProject(rw http.ResponseWriter, req *http.Request) {
 		writeResponse(rw, http.StatusBadRequest, fmt.Sprintf("unexpected error %v", err))
 		return
 	}
-	writeResponse(rw, http.StatusOK, project)
+	encoded, _ := json.Marshal(project)
```

Comment:
Also, if you make this function a `s *Service` function, you can use `s.gceDefaultProjectGetter` to retrieve the default project. That way you can "easily" add tests for this.